vendor/engines/bulk_email/config/routes.rb

```diff
@@ -3,5 +3,7 @@
     get "bulk_email", to: "bulk_email/bulk_email#search"
     post "bulk_email", to: "bulk_email/bulk_email#create"
     post "bulk_email/deliver", to: "bulk_email/bulk_email#deliver"
+    get "bulk_email/history", to: "bulk_email/bulk_email#history"
+    get "bulk_email/job/:id", to: "bulk_email/bulk_email#job", as: "bulk_email_job"
```

Comment:
The BulkEmailController has started to get pretty un-RESTful. Would it be worth splitting these out into `index` and `show` methods on a BulkEmail::JobsController?